pkg/sloop/webserver/webfiles/index.html

```diff
@@ -16,6 +16,9 @@
     <title>Sloop{{if (ne .CurrentContext "")}} - {{.CurrentContext}}{{end}}</title>
     <link rel="stylesheet" type="text/css" href="webfiles/sloop.css">
     <link rel='shortcut icon' type='image/x-icon' href='webfiles/favicon.ico'/>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+          integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
```

Comment:
As there are UI changes you would need to update pkg/sloop/webserver/bindata.go  to update binary data. Please refer to this PR https://github.com/salesforce/sloop/pull/138/files and ReadMe https://github.com/salesforce/sloop#updating-webfiles-folder

Reply:
@sana-jawad Thanks for the info, I noticed in readme stating run 'go-bindata -o bindata.go webfiles/' before submit a PR, I am not quite sure if it matters if I run it now(PR already submitted), and all I need to do is to include the updated bindata.go file in the PR, right?

Reply:
You can include the updated binddata.go in this PR

Reply:
Thanks @sana-jawad. Updated: include bindata.go in the PR